src/screens/Tank/SettleBalanceConfirm/SettleBalanceConfirm.js

```diff
@@ -105,7 +105,7 @@ class SettleBalanceConfirm extends React.Component<Props, State> {
     // add unsettled amounts to balances
     const combinedBalances = Object.keys(balances)
       .reduce((memo, assetName) => {
-        const balanceData: AssetBalance = balances[assetName];
+        const balanceData: WalletAssetBalance = balances[assetName];
         let balance = new BigNumber(balanceData.balance);
```

Comment:
Wrapping in BigNumber should no longer be needed, since balance.balance should now be BigNumber itself.

Reply:
Eh, just noticed that WalletAssetBalance still keeps balance as string. Pls ignore this comment